css/style.css

```diff
@@ -0,0 +1,99 @@
+body {
+  background-color: #EEE;
+  padding-left: 30px;
+  padding-right: 30px;
```

Comment:
This and other rules through out can be simplified using the shorthand value notation for the rule (https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties). 🔨